src/SourceBuild/content/repo-projects/source-build-externals.proj

```diff
@@ -1,9 +1,2 @@
-<Project>
-  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
-
-  <PropertyGroup>
-    <BuildCommand>$(StandardSourceBuildCommand) $(StandardSourceBuildArgs)</BuildCommand>
-  </PropertyGroup>
-
-  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
```

Comment:
FWIW all repos should really have a dependency on Arcade except the ones the use the bootstrap aracde.  Same is true with SBRP